src/action/interactive.rs

```diff
@@ -82,6 +82,9 @@ where
     pub suggestion: &'s Suggestion<'t>,
     /// The content the user provided for the suggestion, if any.
     pub custom_replacement: String,
+    pub i_custom_replacement_end_index: (u16, u16),
+    pub i_custom_replacement_start_column: u16,
+    pub i_cursor_index: u16,
```

Comment:
I am pretty sure not all of them are needed, are they? Start index is implicit, so only a length is necessary which is implicit by the `custom_replacement`

Reply:
`i_cursor_index` is used to track the position of cursor in `insert mode`.
`i_custom_replacement_start_column` I created it to have an abstraction over the gap provided by `  »`. It can be deleted if you think it is a good turn.

Reply:
Currently all of the offsets are hard-coded, and in any case it would be constant which then should be in all occassions.